### PR TITLE
Added Optional ApiKey as a header

### PR DESCRIPTION
**Pull request by mkotsollaris**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,65 +17,65 @@
   ],
```
Comment (kateika) [inline package.json]: why this file has changed? what this update affects functionality?

Comment (mkotsollaris) [reply package.json]: I ran npm-update to update some of our packages.

File: package.json [hunk 2]
```diff
@@ -17,65 +17,65 @@
...
   "dependencies": {
```
Comment (kateika) [inline package.json]: next time it is better to create different PR so we can easily revert the change if it causes any problem
